package/ntrip_daemon/src/ntrip_daemon.c

```diff
@@ -87,87 +87,76 @@ static int parse_options(int argc, char *argv[])
   };
 
   const struct option long_opts[] = {
-    {"ntrip",     no_argument,       0, OPT_ID_MODE_NTRIP},
-    {"settings",  no_argument,       0, OPT_ID_MODE_SETTINGS},
-    {"reconnect", no_argument,       0, OPT_ID_MODE_RECONNECT},
-    {"file",      required_argument, 0, OPT_ID_FILE},
-    {"username",  required_argument, 0, OPT_ID_USERNAME},
-    {"password",  required_argument, 0, OPT_ID_PASSWORD},
-    {"url  ",     required_argument, 0, OPT_ID_URL},
-    {"interval",  required_argument, 0, OPT_ID_INTERVAL},
-    {"rev1gga",   required_argument, 0, OPT_ID_REV1GGA},
-    {"debug",     no_argument,       0, OPT_ID_DEBUG},
+    {"ntrip", no_argument, 0, OPT_ID_MODE_NTRIP},
+    {"settings", no_argument, 0, OPT_ID_MODE_SETTINGS},
+    {"reconnect", no_argument, 0, OPT_ID_MODE_RECONNECT},
+    {"file", required_argument, 0, OPT_ID_FILE},
+    {"username", required_argument, 0, OPT_ID_USERNAME},
+    {"password", required_argument, 0, OPT_ID_PASSWORD},
+    {"url  ", required_argument, 0, OPT_ID_URL},
+    {"interval", required_argument, 0, OPT_ID_INTERVAL},
+    {"rev1gga", required_argument, 0, OPT_ID_REV1GGA},
+    {"debug", no_argument, 0, OPT_ID_DEBUG},
```

Comment:
Don't really like that we're losing the ability to align tables by columns... maybe there's a "no format" program that can be applied for these?